packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5726,7 +5798,10 @@ export default class Meeting extends StatelessWebexPlugin {
       return this.enqueueMediaUpdate(MEDIA_UPDATE_TYPE.UPDATE_MEDIA, options);
     }
 
-    if (this.isMultistream && receiveShare !== undefined) {
+    if (
+      this.isMultistream &&
+      (shareAudioEnabled !== undefined || shareVideoEnabled !== undefined)
+    ) {
       throw new Error(
         'toggling receiveShare in a multistream meeting is not supported, to control receiving screen share call meeting.remoteMediaManager.setLayout() with appropriate layout'
```

Comment:
the error message needs to be updated - it still references non non-existent `receiveShare` from options argument

Reply:
Done.